docs/scripts/api/buildComponentsDocumentation.ts

```diff
@@ -229,6 +229,61 @@ function findXDemos(
     });
 }
 
+function getPackages(filename: string): { exportPackage: string; reExportPackages: string[] } {
```

Comment:
Maybe be extra explicit

```suggestion
function getPackagesFromFilename(filename: string): { exportPackage: string; reExportPackages: string[] } {
```

Reply:
I went to a solution closer to the core PR:
https://github.com/mui/material-ui/pull/38297/files?file-filters%5B%5D=.js&file-filters%5B%5D=.ts&show-viewed-files=true#diff-eef575d465fe9894710e7eb399c76ad3e3900a8cbb6d374be7f303c4b6c6ef51